docs/features.md

```diff
@@ -0,0 +1,88 @@
+# Features
```

Comment:
Maybe some others - but either deprecated in SaaS, trying to deprecate or have never shipped or been supported on-prem (tokens, etc)